Source/Misc/config.cpp

```diff
@@ -0,0 +1,29 @@
+//Fluffy: For loading various options from the game's config file
+
+#include "..\all.h"
+#include "../../3rdParty/Storm/Source/storm.h"
```

Comment:
inconsistent usage of backslash / forward slash ;)

Reply:
Fixed

Reply:
not only in this place :P